lemur/views.py

```diff
@@ -479,4 +487,13 @@ def inject_patterns():
                 pattern_for_value_range=('[0-9]{1,10}[.]?[0-9]{0,10}'
                                          '-[0-9]{1,10}[.]?[0-9]{0,10}'),
                 pattern_for_value_range_hint=('valueRange should be'
-                                              'in the format:0.3-6.5'))
+                                              'in the format:0.3-6.5'),
+                pattern_for_class_time='[0-9]{2,4}[a-zA-Z]{4,7}',
+                pattern_for_class_time_hint=('Class Time is a combination',
+                                             'of year and semester. e.g. ',
+                                             '16fall'),
+                pattern_for_student_name='([0-9a-zA-Z\-]*)((,[0-9a-zA-Z\-]*)*)',
```

Comment:
In the latest version I have changed this field to be a list of student's ids that can be selected rather than typing to avoid typo